staff/manuel-eranueva/strings/concate copy.js

```diff
@@ -0,0 +1,17 @@
+delete String.prototype.concat
+
+function concat(string1, string2) {
+    var string3 = ''
+    if (string1 == Number && string2 == Number) {
+        string3 = string1.toString() + string2
+    } else {
```

Comment:
Check the behaviour of concat() when we try to concatenate to numbers: 

```
var firstNumber = 10;
var secondNumber = 20;
var concatResult = firstNumber.concat(secondNumber)

console.log(concatResult);
```

It should throw an error